health/notifications/health_alarm_notify.conf

```diff
@@ -189,21 +189,22 @@ aws=""
 # EMAIL_SENDER="User Name <user@domain>"
 # EMAIL_SENDER="'User Name' <user@domain>"
 # EMAIL_SENDER="\"User Name\" <user@domain>"
-EMAIL_SENDER=""
+EMAIL_SENDER="Netdata System Alert <netdata@brunocruz-dev.go-contact.com>"
```

Comment:
same here, the default config must be empty. You can provide a commented example.